src/coreclr/pal/src/exception/signal.cpp

```diff
@@ -206,8 +212,10 @@ BOOL SEHInitializeSignals(CorUnix::CPalThread *pthrCurrent, DWORD flags)
         }
 
         g_stackOverflowHandlerStack = (void*)((size_t)g_stackOverflowHandlerStack + stackOverflowStackSize);
+#endif // HAVE_MACH_EXCEPTIONS
     }
 
+#if !HAVE_MACH_EXCEPTIONS
```

Comment:
Just below this is code setting SIGPIPE to  SIG_IGN.

There already exists code to do this in machexception.cpp. Could we remove that code and un-ifdef the code here, to consolidate all signal handling in this one file? 

https://github.com/dotnet/runtime/blob/d582900145d6ecf752f32b272c6b87c58f4470bd/src/coreclr/pal/src/exception/machexception.cpp#L1419